test_rails_4_app/spec/rake_task_spec.rb

```diff
@@ -1,18 +1,14 @@
 require 'rails_helper'
+require 'rake'
 
 describe "rake active_mocker:build" do
+  before { SampleApp::Application.load_tasks }
   it "builds the mocks from the model dir" do
-    expect(File.exist? mock_path).to eq true
-    expect(File.exist? File.join(mock_path, 'user_mock.rb')).to eq true
-    expect(File.exist? File.join(mock_path, 'micropost_mock.rb')).to eq true
+    expect(File.exist? mock_path                                   ).to eq true
+    expect(File.exist? File.join(mock_path, 'user_mock.rb')        ).to eq true
+    expect(File.exist? File.join(mock_path, 'micropost_mock.rb')   ).to eq true
     expect(File.exist? File.join(mock_path, 'relationship_mock.rb')).to eq true
-    result = File.exist?(File.join(mock_path, 'api/customer_mock.rb'))
-    puts ""
-    puts mock_path
-    system "ls #{mock_path}"
-    puts File.join(mock_path, 'api')
-    system "ls #{File.join(mock_path, 'api')}"
-    expect(result).to eq true
+    expect(File.exist? File.join(mock_path, 'api/customer_mock.rb')).to eq true
```

Comment:
Add parentheses to nested method call `File.exist? File.join(mock_path, 'api/customer_mock.rb')`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.